Libplanet.Benchmarks/Store.cs

```diff
@@ -0,0 +1,147 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using BenchmarkDotNet.Attributes;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Libplanet.Store;
+using Libplanet.Tests;
+using Libplanet.Tests.Common.Action;
+using Libplanet.Tests.Store;
+using Libplanet.Tx;
+
+namespace Libplanet.Benchmarks
+{
+    public class Store
+    {
+        private static readonly ImmutableArray<Block<DumbAction>> Blocks;
+        private static readonly int BlockCount;
+        private static readonly ImmutableArray<Transaction<DumbAction>> Txs;
+        private static readonly int TxsCount;
+        private IStore _store = null;
+
+        static Store()
+        {
+            List<Block<DumbAction>> blocks = new List<Block<DumbAction>>();
+            List<Transaction<DumbAction>> txs = new List<Transaction<DumbAction>>();
+            Block<DumbAction> block = TestUtils.MineGenesis<DumbAction>();
+            blocks.Add(block);
+            PrivateKey key = new PrivateKey();
```

Comment:
Trivial, just for consistency.

```suggestion
            var key = new PrivateKey();
```